packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebView.java

```diff
@@ -42,7 +44,17 @@
     DisplayManager displayManager =
         (DisplayManager) context.getSystemService(Context.DISPLAY_SERVICE);
     displayListenerProxy.onPreWebViewInitialization(displayManager);
-    webView = new InputAwareWebView(context, containerView);
+
+    Context activityContext = context;
+    Context appContext = context.getApplicationContext();
+    if (appContext instanceof FlutterApplication) {
+      Activity currentActivity = ((FlutterApplication) appContext).getCurrentActivity();
+      if (currentActivity != null) {
+        activityContext = currentActivity;
+      }
+    }
+
+    webView = new InputAwareWebView(activityContext, containerView);
```

Comment:
We must instantiate the WebView with the presentation context, the activity context isn't tuned for the VD size, doesn't use our custom window manager, and won't use our IMM wrapper.

Reply:
Thanks @amirh , another person have made another PR: https://github.com/flutter/plugins/pull/2081/commits/8fc8eb62edf39c1f8db8282c1a1d9661c5f734e2

If I understand your comment correctly, this approach is not right as well. Do you confirm ?

Reply:
Yeap, I confirm uldaman's PR quoted in previous comment crashes at least on
- API 23 real device
- API 24 emulator